### PR TITLE
[FIX] correction to configuration details

### PR DESCRIPTION
**Pull request by sindhu-106**
Signed-off-by: sindhuja <sindhuja.gajula@intel.com>

### CODE REVIEW

File: config-files/FabricTelemetryProvider.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
-  "connectAddress": "sms01-nmn.local",
-  "connectPort": 8080,
-  "urlPath": "/apis/sma-telemetry/v1/stream/cray-fabric-telemetry",
+  "fullUrl": "http://api-gw-service-nmn.local:8080/apis/sma-telemetry/v1/stream/cray-fabric-telemetry?stream_id=dai-fabric-telemetry",
```
Comment (yhuan47) [inline config-files/FabricTelemetryProvider.json:2]: Port 8080 is from old version of Admin Guide. It is not required in new version. Paul said it is using default port.

File: config-files/HWInvDiscoveryConfig.json [hunk 2]
```diff
@@ -13,27 +11,27 @@
     },
     "requester": {
       "initiateDiscovery": {
-        "endpoint": "http://sms01-nmn.local:8080",
+        "endpoint": "http://api-gw-service-nmn.local:8080",
```
Comment (yhuan47) [inline config-files/HWInvDiscoveryConfig.json:14]: Port is not required for all APIs.

File: config-files/ProviderMonitoringNetworkForeignBus.json [hunk 3]
```diff
@@ -1,119 +1,54 @@
 {
   "providerClassMap": {
-    "environmentalData": "com.intel.dai.monitoring.EnvironmentalProviderForeignBus",
-    "rasData": "com.intel.dai.monitoring.RasEventProviderForeignBus"
+    "environmentalData": "com.intel.dai.monitoring.EventSimEnvironmentalProviderForeignBus",
+    "rasData": "com.intel.dai.monitoring.EventSimRasEventProviderForeignBus"
```
Comment (paul-amonson) [inline config-files/ProviderMonitoringNetworkForeignBus.json:4]: For the in house system you should be using the non-EventSim providers

**Review by yhuan47 — DISMISSED** [review]

File: config-files/ProviderMonitoringNetworkForeignBus.json [hunk 4]
```diff
@@ -1,119 +1,54 @@
 {
   "providerClassMap": {
-    "environmentalData": "com.intel.dai.monitoring.EnvironmentalProviderForeignBus",
+    "environmentalData": "com.intel.dai.monitoring.SensorMetaDataForeignBus",
```
Comment (paul-amonson) [inline config-files/ProviderMonitoringNetworkForeignBus.json:3]: Oops. Wrong class. should be com.intel.dai.monitoring.EnvironmentalProviderForeignBus

**Pull request merged**